devtools-ci-index.yaml

```diff
@@ -4542,6 +4542,28 @@
             git_repo: f8a-stacks-report
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: f8a-stacks-report
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-analytics-bigquery-manifests-job
+            saas_git: saas-analytics
+            prj_name: bayesian-preview
+            saas_service_name: f8a-bq-manifests-job
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            timeout: '20m'
+            extra_target: rhel
+        - '{ci_project}-{git_repo}-fabric8-analytics':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-analytics-bigquery-manifests-job
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
+            registry: 'quay.io'
+            image_name: 'openshiftio/fabric8-analytics-f8a-bq-manifests-job'
+        - '{ci_project}-{git_repo}-fabric8-analytics-pylint':
+            git_repo: fabric8-analytics-bigquery-manifests-job
```

Comment:
this will try to run `cico_run_pylint.sh` which does not exist:
https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L725-L730

Reply:
added now